test/integration/Identity.test.ts

```diff
@@ -335,13 +335,20 @@ describe('A Solid server with IDP', (): void => {
     });
   });
 
-  describe('openid-configuration', (): void => {
-    it('should contain solid_oidc_supported key.', async(): Promise<void> => {
+  describe('setup', (): void => {
```

Comment:
That's a weird change?

Reply:
Was just a way to group all independent tests related to the IDP. All the describe blocks above are part of the registration/login procedure.